diff -Naur libmodplug-0.8.7.orig/src/libmodplug/stdafx.h libmodplug-0.8.7/src/libmodplug/stdafx.h
--- libmodplug-0.8.7.orig/src/libmodplug/stdafx.h	Mon Apr 20 07:09:16 2009
+++ libmodplug-0.8.7/src/libmodplug/stdafx.h	Fri Mar  6 20:14:33 2015
@@ -36,9 +36,20 @@
 #include <stdio.h>
 #include <malloc.h>
 
+#pragma warning(disable:4996)
+
+typedef unsigned __int32 uint32_t;
+typedef          __int32  int32_t;
+typedef unsigned __int16 uint16_t;
+typedef          __int16  int16_t;
+typedef unsigned __int8   uint8_t;
+typedef          __int8    int8_t;
+
 #define srandom(_seed)  srand(_seed)
 #define random()        rand()
 #define sleep(_ms)      Sleep(_ms)
+#define strncasecmp     strnicmp
+#define strcasecmp      stricmp
 
 inline void ProcessPlugins(int n) {}
 
diff -Naur libmodplug-0.8.7.orig/src/sndfile.cpp libmodplug-0.8.7/src/sndfile.cpp
--- libmodplug-0.8.7.orig/src/sndfile.cpp	Wed Apr 22 07:47:42 2009
+++ libmodplug-0.8.7/src/sndfile.cpp	Fri Mar  6 20:17:46 2015
@@ -6,8 +6,8 @@
 */
 
 #include <math.h> //for GCCFIX
-#include <libmodplug/stdafx.h>
-#include <libmodplug/sndfile.h>
+#include "libmodplug/stdafx.h"
+#include "libmodplug/sndfile.h"
 
 #define MMCMP_SUPPORT
 
diff -Naur libmodplug-0.8.7.orig/src/sndmix.cpp libmodplug-0.8.7/src/sndmix.cpp
--- libmodplug-0.8.7.orig/src/sndmix.cpp	Mon Apr 20 05:00:54 2009
+++ libmodplug-0.8.7/src/sndmix.cpp	Fri Mar  6 20:17:41 2015
@@ -4,8 +4,8 @@
  * Authors: Olivier Lapicque <olivierl@jps.net>
 */
 
-#include <libmodplug/stdafx.h>
-#include <libmodplug/sndfile.h>
+#include "libmodplug/stdafx.h"
+#include "libmodplug/sndfile.h"
 #include "tables.h"
 
 #ifdef MODPLUG_TRACKER
diff -Naur libmodplug-0.8.7.orig/src/tables.h libmodplug-0.8.7/src/tables.h
--- libmodplug-0.8.7.orig/src/tables.h	Mon Apr 20 05:00:54 2009
+++ libmodplug-0.8.7/src/tables.h	Fri Mar  6 20:17:34 2015
@@ -4,8 +4,8 @@
  * Authors: Olivier Lapicque <olivierl@jps.net>
  */
 
-#include <libmodplug/stdafx.h>
-#include <libmodplug/sndfile.h>
+#include "libmodplug/stdafx.h"
+#include "libmodplug/sndfile.h"
 
 #ifndef MODPLUG_FASTSOUNDLIB
 //#pragma data_seg(".tables")
